meta/types.go

```diff
@@ -30,7 +30,8 @@ var Types = []TypeInfo{
 	TypeInfo{Prefix: "NullString", Literal: "sql.NullString"},
 	TypeInfo{Prefix: "NullTime", Literal: "NullableTime"}, // TODO(shutej): test
 	TypeInfo{Prefix: "String", Literal: "string"},
-	TypeInfo{Prefix: "Time", Literal: "time.Time"}, // TODO(shutej): test
+	TypeInfo{Prefix: "StringArray", Literal: "[]string"}, // TODO(benbardin): test
```

Comment:
it doesn't seem to know that a varchar[] in a DB is something it can handle, is there another hook i need to add this to?